controller/raft/hclog_adapter.go

```diff
@@ -0,0 +1,231 @@
+package raft
+
+import (
+	"fmt"
+	"github.com/hashicorp/go-hclog"
```

Comment:
Is there a reason we need yet-another-logger?

Reply:
hclog is what the hashicorp raft library uses. This is an adapter so the raft code logging will get converted to logrus and we can hopefully have consistent logging output